lxd/storage_volumes_backup.go

```diff
@@ -665,8 +664,8 @@ func storagePoolVolumeTypeCustomBackupPost(d *Daemon, r *http.Request) response.
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["volume"] = []string{volumeName}
+	resources := map[string][]api.URL{}
+	resources["volume"] = []api.URL{*api.NewURL().Path(version.APIVersion, "storage-pools", poolName, "volumes", volumeTypeName, volumeName, "backups", backup.Render().Name)}
```

Comment:
This should be `backups` like in storagePoolVolumeTypeCustomBackupsPost

Reply:
for instance rename, we use the old name in resources

Reply:
OK, lets keep it all consistent for now and stick with that approach.